app/styles/site-views/search-list.scss

```diff
@@ -0,0 +1,80 @@
+$app-qgreen: rgba(98,144,128,0.25);
+$app-halfgreen: rgba(98,144,128,0.5);
+$app-lightgray: rgb(239,239,239);
+
+.centerdiv-list-wrapper {
+  background-color: $app-lightgray;
+}
+
+.centereddiv-list {
+
+  height: auto;
+  width:auto;
+
+}
+
+.centereddiv-list-dropdown {
+  height: auto;
+  width: auto;
+  align-content: left;
+
+
+}
+
+.list-footer {
+  font-size: 15px;
+  margin-top: 20px;
+}
+
+.list-toprow {
+  margin-top: 5px;
+
```

Comment:
same as a above -.list-top-row (these are custom classes right? not overriding classes in bootstrap?)

Reply:
yes custom classes.

Reply:
i have changed classes with prefix such as landing, list or form.